### ISSUE TITLE
No on-focus indicator on buttons used for opening modals

### ISSUE THREAD

**Issue by ayanchoudhary** [cc-archive/vocabulary-legacy#562]
## Description

Some of the button components for opening modals don't have an on-focus indicator.

## Reproduction

1. Try navigating through the keyboard to the *report this content* or *safe browsing* buttons.
2. There is no indicator that the focus is on these buttons.

4. See error.

## Expectation

There should be a focus indicator, ideally a border around the element.

## Screenshots

![CC Search - Brave 05-08-2020 18_44_31](https://user-images.githubusercontent.com/45792768/89418660-fc166080-d74d-11ea-8d2f-cdbdf0bb72d6.png)
![CC Search - Brave 05-08-2020 18_45_31](https://user-images.githubusercontent.com/45792768/89418837-37b12a80-d74e-11ea-96dc-c92d0568f471.png)

## Environment

 - Device: (_eg._ iPhone Xs; laptop)
 - OS: (_eg._ iOS 13.5; Fedora 32)
 - Browser: (_eg._ Safari; Firefox)
 - Version: (_eg._ 13; 73)
 - Other info: (_eg._ display resolution, ease-of-access settings)

## Resolution

- [ ] I would be interested in resolving this bug.

**Comment by rochisha0** [comment]
Can I take this issue? @AyanChoudhary

**Comment by ayanchoudhary** [comment]
We might have to wait for @panchovm to decide how the indicator will look. Once that is figured out you can go ahead.

**Comment by fcoveram** [comment]
This ticket will need to wait until we have decided on the final version of brand colors/accessible version. Unfortunately, this change depends on brand/org decisions and it will take more time.

**Comment by zackkrida** [comment]
Hover state should be an underline for these buttons, and added to vocabulary.

**Comment by dhruvkb** [comment]
Just wanted to say that on macOS, certain settings might affect how this behaviour works, further complicated by the fact that not all browsers honour this OS setting.
![Screenshot 2020-08-31 at 11 40 06 AM](https://user-images.githubusercontent.com/16580576/91688051-b9924900-eb7e-11ea-858e-8a70de141e4d.png)
